docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+services:
+  db:
+    image: postgres
+    environment:
+      - POSTGRES_USER=docker
+      - POSTGRES_PASSWORD=docker
+      - POSTGRES_DB=siglateste
```

Comment:
I changed db names on my PR, should I rollback to the other name or we fix it later?

Reply:
No, continue with your changes, I'll update this with yours.